.github/workflows/create-urgent-hotfix-img.yml

```diff
@@ -0,0 +1,301 @@
+          secrets: |
+            secret/data/github.com/organizations/camunda NEXUS_PSW | ci-account-password;
+            secret/data/github.com/organizations/camunda NEXUS_USR | ci-account-username;
+
+      - name: Log into Harbor
+        uses: docker/login-action@v3
+        with:
+            registry: registry.camunda.cloud
+            username: ${{ steps.secrets.outputs.ci-account-username }}
+            password: ${{ steps.secrets.outputs.ci-account-password }}
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Setup Java
+        uses: actions/setup-java@v4
+        with:
+          distribution: 'temurin'
+          java-version: '21'
+
+      - name: Build Maven artifacts
+        run: |
+          echo "🔨 Building Maven artifacts for ${{ matrix.component }}..."
+
+          case "${{ matrix.component }}" in
+            "zeebe"|"camunda"|"operate"|"tasklist")
+              echo "📦 Building ${{ matrix.component }} distribution tarball..."
+              ./mvnw clean package -T1C -DskipTests -DskipChecks -am -pl dist
+              ;;
+            "optimize")
+              echo "📦 Building optimize distribution tarball..."
+              if [[ ! -f "optimize.Dockerfile" ]]; then
+                echo "❌ ERROR: Optimize not available in this version (only supported from 8.8+)"
+                exit 1
+              fi
+              ./mvnw clean package -T1C -DskipTests -DskipChecks -am -pl optimize-distro
+              ;;
+            *)
+              echo "❌ Unknown component: ${{ matrix.component }}"
+              exit 1
+              ;;
+          esac
+
+      - name: Configure Docker build args
+        id: build-args
+        run: |
+          BUILD_DATE=$(date -u +'%Y-%m-%dT%H:%M:%SZ')
+
+          if [[ "${{ matrix.component }}" == "optimize" ]]; then
+            # Optimize uses wildcards to match whatever version Maven builds
+            BUILD_ARGS="VERSION=*
+            DATE=$BUILD_DATE
+            REVISION=${{ steps.resolve.outputs.ref }}
+            DISTRO=production
+            ARTIFACT_PATH=./optimize-distro/target"
+          else
+            # Zeebe, Camunda, Operate, and Tasklist all use DISTBALL with wildcard
+            BUILD_ARGS="VERSION=${{ matrix.version }}
+            DATE=$BUILD_DATE
+            REVISION=${{ steps.resolve.outputs.ref }}
+            DISTBALL=dist/target/camunda-zeebe-*.tar.gz"
+          fi
+
+          # Use printf to preserve multiline format and escape newlines for output
+          {
+            echo "args<<EOF"
+            echo "$BUILD_ARGS"
+            echo "EOF"
+          } >> "$GITHUB_OUTPUT"
+
+      - name: Build and push Docker image
+        uses: docker/build-push-action@v6
+        with:
+          context: .
+          file: ${{ matrix.component == 'zeebe' && 'Dockerfile' || format('{0}.Dockerfile', matrix.component) }}
+          platforms: ${{ env.PLATFORMS }}
+          push: true
+          sbom: true
```

Comment:
:x: 

why SBOM is being pushed here?I don't see that being done in docker places using docker/build-push-action.